utils/Parser/APIResponseParser.tsx

```diff
@@ -415,15 +415,42 @@ export function parseSkyblockProfile(profile): SkyblockProfile {
 
 export function parseCraftingRecipe(recipe): CraftingRecipe {
     return {
-        A1: recipe.A1.split(':')[0],
-        A2: recipe.A2.split(':')[0],
-        A3: recipe.A3.split(':')[0],
-        B1: recipe.B1.split(':')[0],
-        B2: recipe.B2.split(':')[0],
-        B3: recipe.B3.split(':')[0],
-        C1: recipe.C1.split(':')[0],
-        C2: recipe.C2.split(':')[0],
-        C3: recipe.C3.split(':')[0]
+        A1: {
+            tag: recipe.A1.split(':')[0],
+            count: recipe.A1.split(':')[1]
```

Comment:
Why isn't this a constructor method and duplicated 9 times?

Reply:
c8259897a4d50febcb748041f71ae7635dcdb3c9